python/tool/sphinx/rules.bzl

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+def _sphinx_docs_impl(ctx):
+    driver_package = ctx.actions.declare_directory("driver-package")
```

Comment:
I think similarly to `nodejs` tools, we can remove references to `driver` in here? These mentions are all for intermediate files and we can name them accordingly (eg. `driver_package` -> `python_package`)